web/test/pages/profile/profile-shared.test.tsx

```diff
@@ -400,4 +400,30 @@ describe("profile - shared", () => {
       ).not.toBeInTheDocument();
     });
   });
+
+  describe("profile error alert", () => {
+    it.each(["STARTING", "FOREIGN"])(
```

Comment:
[sand] I think we can include the `owning` tests here as well. Can we create a refactor ticket for this?